### PR TITLE
Add Carousel

### PR DESCRIPTION
**Pull request by IkaPurnama**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -26,6 +26,30 @@
             </li>
         </ul>
     </nav>
+
+    <!-- Carousel Section -->
+    <div id="carouselExample" class="carousel slide">
+        <div class="carousel-inner">
+          <div class="carousel-item active">
+            <img src="..." class="d-block w-100" alt="...">
```
Comment (joviantomd) [inline index.html]: Add some image

**Review by joviantomd — CHANGES_REQUESTED** [review]

File: index.html [hunk 2]
```diff
@@ -26,6 +26,30 @@
...
+          </div>
+          <div class="carousel-item">
+            <img src="..." class="d-block w-100" alt="...">
```
Comment (Justintnwj) [inline index.html]: Please drop Picture Pinguine

**Review by Justintnwj — CHANGES_REQUESTED** [review]

**Review by joviantomd — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (IkaPurnama) [reply index.html]: done

File: index.html (hunk 2 above)
Comment (IkaPurnama) [reply index.html]: Done

**Review by Justintnwj — APPROVED** [review]

**Pull request merged**